added args to function definition and tol argument to regularization_…

conleyst opened the pull request:
I added the arguments to the headers of the function definitions and added an argument for tolerance in the regularization_plot function. Tolerance arg will be used as a standard, where coefficients less than it will be treated as zero.

conleyst committed:
added args to function definition and tol argument to regularization_plot

tylercroberts commented on ezmodel/ezmodel.py:
```diff
@@ -36,14 +36,16 @@ def regularization_plot():
-        alpha (): Penalty constant multiplying the regularization term. Larger value corresponds to stronger
+        alpha (float or list): Penalty constant multiplying the regularization term. Larger value corresponds to stronger
                   regularization. Can be list or float.
 
```
Maybe just be a bit more specific about what the returns are for float and list inputs to `alpha`

conleyst committed:
Updated docstring of regularization_plot and removed coerce

Updated the docstring for `regularization_plot` so that it's clearer exactly what is returned based on alpha type and what the plot shows.

Removed the coerce function and its docstring.

conleyst commented:
Updated docstring and removed `coerce` function.
